app/views/users/show.html.erb

```diff
@@ -142,8 +142,11 @@
   <% repos = params[:show].eql?('makes') ? @makes : @repositories %>
   <% repos.each do |repository| %>
     <div class="repository-container">
-
-      <div class="repo-display-wrapper" <%= "style=background-image:url('#{@photos[repository.id].image.url}')"  %> >
+      <% if repository.photos.exists? %>
```

Comment:
I realized if there is no photo, the black background will show up for the repositories. So I didn't see a point in adding an icon or a default photo to them; instead I used the if statement to try to show the photo only if one exists.